queries.sql

```diff
@@ -21,3 +21,74 @@ WHERE name != 'Gabumon';
 
 SELECT * FROM animals
 WHERE weight_kg between 10.4 and 17.3;
+
+-- changing the species 
+
+update animals
+set species = 'digimon'
+where name like '%mon';
+
+update animals
+set species = 'pokemon'
+where species is null;
+
+-- deleting animals and rollback
```

Comment:
- [ ] Please make sure to add the above SQL statements in a transaction with proper `BEGIN`, `COMMIT` and `ROLLBACK` and also make sure to verify that the change was made before rolling back and after making a commit with `SELECT` statements.